src/app_service/service/community_tokens/async_tasks.nim

```diff
@@ -227,20 +236,86 @@ type
 const fetchCollectibleOwnersTaskArg: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
   let arg = decode[FetchCollectibleOwnersArg](argEncoded)
   try:
-    let response = collectibles.getCollectibleOwnersByContractAddress(arg.chainId, arg.contractAddress)
+    var response = collectibles.getCollectibleOwnersByContractAddress(arg.chainId, arg.contractAddress)
```

Comment:
Is check `if not response.error.isNil:` missing here?

Reply:
rpc macro inside is using callPrivateRPC which will raise an exception if there is a response error